### PR TITLE
Added kafka_redis and git clone notebooks

### PR DESCRIPTION
**Pull request by GiladShapira94**
Added two notebooks one for git clone and run testing and the second one for test feature store with Kafka and Redis

### CODE REVIEW

File: git_clone_test/load_project_test.ipynb [hunk 1]
```diff
@@ -1,9 +1,22 @@
 {
```
Comment (MichaelSkralivetsky) [inline git_clone_test/load_project_test.ipynb:1]: <p>please do not use repos owned by your private github user GiladShapira94, could be repo in some generic org like iguazio or mlrun instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (MichaelSkralivetsky) [inline git_clone_test/load_project_test.ipynb:1]: <p>mount_s3 is not safe and seems won't be ever safe.</p><p>you just exposed by yaml print some AWS credentials in this PR, please ask devops to delete exposed AWS_ACCESS_KEY_ID</p><p>and remove the print</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: kafka_redis_fs/fs-kafka-redis.ipynb [hunk 2]
```diff
@@ -1,21 +1,42 @@
 {
```
Comment (MichaelSkralivetsky) [inline kafka_redis_fs/fs-kafka-redis.ipynb:1]: <p>why this was changed to load_project ? please use get_or_create_project as before</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (GiladShapira94) [reply kafka_redis_fs/fs-kafka-redis.ipynb:1]: <p>Changed</p>

File: git_clone_test/load_project_test.ipynb (hunk 1 above)
Comment (GiladShapira94) [reply git_clone_test/load_project_test.ipynb:1]: <p>Deleted all non relevant prints</p>

File: kafka_redis_fs/fs-kafka-redis.ipynb (hunk 2 above)
Comment (MichaelSkralivetsky) [inline kafka_redis_fs/fs-kafka-redis.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    resp
</pre><p>last thing - at the end of the notebook can u please add purge_targets() to your feature set so there won't be any garbage left in our centralized redis db after the run. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (GiladShapira94) [reply kafka_redis_fs/fs-kafka-redis.ipynb:1]: Added purge_targets()

**Pull request merged**